package.json

```diff
@@ -18,6 +18,7 @@
     "async": "~0.9.0",
     "babel-runtime": "^5.5.4",
     "bignumber.js": "^2.0.3",
+    "es6-promisify": "^2.0.0",
```

Comment:
I'm not sure, is this what you mean? http://stackoverflow.com/questions/21846390/getting-a-promises-value-via-yield-co